### PR TITLE
update libre-graph-api to v1.0

### PR DESCRIPTION
**Pull request by fschade**
## Description
update libre-graph-api to v1.0

### CODE REVIEW

**Review by dschmidt — APPROVED** [review]

**Review by JammingBen — APPROVED** [review]

**Review by kulmann — APPROVED** [review]

File: changelog/unreleased/change-update-libregraph [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Change: Update libre-graph-api to v1.0
```
Comment (kulmann) [inline changelog/unreleased/change-update-libregraph:1]: this is type `enhancement` isn't it?

**Review by kulmann — CHANGES_REQUESTED** [review]

File: changelog/unreleased/change-update-libregraph (hunk 1 above)
Comment (fschade) [reply changelog/unreleased/change-update-libregraph:1]: Thought the same, but vue bump is classified as change too.

https://github.com/owncloud/web/blob/master/changelog/unreleased/change-update-vue

should i change it?

Comment (kulmann) [reply changelog/unreleased/change-update-libregraph:1]: but this only adds tag management. so it's an enhancement :-)

Comment (fschade) [reply changelog/unreleased/change-update-libregraph:1]: i change it, can we force merge? don't want to wait for a full ci run (which will be the same as the last)

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/30812/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/30812/tracing/upload-resources-in-personal-space-alice-2022-12-19-03-18-44.zip 


</p></details>
:boom: The e2e-oc10 tests pipeline failed. The build has been cancelled.

**Pull request merged**